pymeasure/instruments/hp/hp34401A.py

```diff
@@ -49,3 +49,229 @@ def __init__(self, adapter, **kwargs):
             asrl={'baud_rate': 9600, 'data_bits': 7, 'parity': 2},
             **kwargs
         )
+
+    function_ = Instrument.control(
+        "FUNC?", "FUNC \"%s\"",
+        """ A string property that controls the measurement function,
```

Comment:
Please follow the current style of code, which is not yet implemented in all instruments: https://pymeasure.readthedocs.io/en/latest/dev/coding_standards.html#docstrings

Especially a docstring (python standard) has to be in imperative Form and we (pymeasure) want to start them with "Control", "Set", or "Measure" to indicate the type of property.

Reply:
I improved the docstrings. Are they fine now from your side?

Reply:
Yes, they are good. Thanks.